doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -73,6 +73,85 @@ See :ref:`rocq_makefile` and :ref:`building_dune`.
       .. rocqdoc::
          -R <PATH> Mod1
 
+System configuration
+--------------------
+
+Running `rocq` (or the `coq` compatibility commands) will fail if it
+cannot find certain expected files (except for subcommands like
```

Comment:
Maybe we should document which files too?

Reply:
That's what the rest of this section does (mostly it's the Prelude.vo)